lib/feature_flagger/model.rb

```diff
@@ -44,13 +52,19 @@ def released_id?(resource_id, *feature_key)
       end
 
       def release_id(resource_id, *feature_key)
-        feature = Feature.new(feature_key, feature_flagger_model_settings.entity_name)
-        FeatureFlagger.control.release(feature.key, resource_id)
+        resource_name = feature_flagger_model_settings.entity_name
+        feature = Feature.new(feature_key, resource_name)
+        FeatureFlagger.control.release(feature.key, resource_id, resource_name)
```

Comment:
can we extract `resource_name` from `feature.key`?
With that, we do not break API compatibility with users who wrongly use FeatureFlagger.control directly (which is the RDSM case).